kolibri/tasks/api.py

```diff
@@ -0,0 +1,67 @@
+import importlib
+from celery import Celery
+from django.core.management import get_commands, call_command
+
+import logging as logger
+
+logging = logger.getLogger(__name__)
+
+
+app = Celery()
+app.config_from_object('kolibri.deployment.default.celeryconfig.default')
+
+
+CALL_COMMAND_SHORTNAME = 'kolibri.call_command'
+
+
+def setup_celery_for_management_commands():
+
+    # import all management commands, so when the worker forks, it has them in memory.
```

Comment:
Would be interested to learn more. Why do they already need to be in memory at the time it forks?